doc/CMakeLists.txt

```diff
@@ -123,21 +134,45 @@ else (DOXYGEN_FOUND)
 endif (DOXYGEN_FOUND)
 
 if (INSTALL_DOCUMENTATION)
-	install (FILES BIGPICTURE.md DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER})
-	install (FILES GOALS.md DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER})
-	install (FILES METADATA.ini DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER})
-	install (FILES SECURITY.md DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER})
-	install (FILES AUTHORS.md DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER})
-	install (FILES CONTRACT.ini DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER})
+	install (
+		FILES BIGPICTURE.md
+		DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER}
+		COMPONENT libelektra4)
+	install (
+		FILES GOALS.md
+		DESTINATION ${TARGET_DOCUMENTATION_TEXT_FOLDER}
+		COMPONENT libelektra4)
+	install (
+		FILES METADATA.ini
```

Comment:
Ahh, good that this finally gets installed! Does the script scripts/kdb/mount-info work? (It should as you use TARGET_DOCUMENTATION_TEXT_FOLDER)

Reply:
There is an issue with it. Although these files are inside the debian package (at /usr/share/doc/elektra/) they somehow doesn't get installed.

Reply:
Seems like dpkg somehow removes these files.. Running dpkg with debug=7200 returns following output:

```
D000010: tarobject ti->name='./usr/share/doc/elektra/NEWS.md' mode=100644 owner=0:0 type=48(-) ti->linkname='' namenode='/usr/share/doc/elektra/NEWS.md' flags=2 instead='<none>'
D000010: filter comparing '/usr/share/doc/elektra/NEWS.md' and '/usr/share/man/*'
D000010: filter comparing '/usr/share/doc/elektra/NEWS.md' and '/usr/share/locale/*/LC_MESSAGES/*.mo'
D000010: filter comparing '/usr/share/doc/elektra/NEWS.md' and '/usr/share/doc/*'
D000010: filter removing ./usr/share/doc/elektra/NEWS.md
D000010: filter comparing '/usr/share/doc/elektra/NEWS.md' and '/usr/share/doc/*/copyright'
D000010: filter comparing '/usr/share/doc/elektra/NEWS.md' and '/usr/share/doc/*/changelog.Debian.*'

```

This also happens on the previous Elektra packages not build with CPack.